src/assets/i18n/en.json

```
@@ -33,6 +33,16 @@
       "ADD_MANAGER": "Add manager",
       "CREATE_GROUP": "Create group",
       "ADD_MEMBER": "Add member"
+    },
+    "RESOURCES": {
+      "TITLE": "Resources",
+      "DELETE": "Remove selected",
+      "TABLE_RESOURCE_ID": "id",
+      "TABLE_RESOURCE_NAME": "Name",
+      "TABLE_FACILITY_NAME": "Facility name",
+      "TABLE_RESOURCE_TAGS": "Tags",
+      "TABLE_RESOURCE_DESCRIPTION": "Description",
+      "TABLE_SEARCH": "Search"
```

```suggestion
      "TABLE_SEARCH": "Filter by name or description"
```

Also, if the text doesn't fit, you can make the input a bit wider.